Монин Константин

Kuvadorian opened the pull request:
[Посмотреть решение](https://hruview.herokuapp.com/Kuvadorian/markup-task-5/master/index.html)

Kuvadorian committed:
Complete

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

msmirnov commented on index.css:
```diff
@@ -0,0 +1,312 @@
+.head
+{
+    font-family: 'PT Sans Narrow', sans-serif;
```
~~Этот шрифт есть не везде. Для кроссбраузерности его нужно отдельно подключить.~~

msmirnov replied:
Это замечание снимается, пропустил в задании

msmirnov commented on index.css:
```diff
@@ -0,0 +1,312 @@
+.head
+{
+    font-family: 'PT Sans Narrow', sans-serif;
+    position: fixed;
```
~~А зачем здесь position:fixed?~~

msmirnov commented on index.css:
```diff
@@ -0,0 +1,312 @@
+    position: fixed;
+    width: 100%;
+    background: white;
+    margin: auto;
```
~~Зачем, если есть position:fixed и width:100% ?~~

msmirnov commented on index.css:
```diff
@@ -0,0 +1,312 @@
+body
+{
+    margin: 0;
+    text-align: center;
```
Лучше не выравнивать текст на всём сайте сразу, а делать это только там, где надо. Например, в модальном окне.

msmirnov commented on index.css:
```diff
@@ -0,0 +1,312 @@
+    font-family: 'PT Sans Narrow', sans-serif;
+    position: fixed;
+    width: 100%;
+    background: white;
```
В body же уже указан цвет

msmirnov commented on index.css:
```diff
@@ -0,0 +1,312 @@
+input
+{
+    margin-top: 75px;
+    display: none;
```
Лучше бы более специфичный селектор указать. Представь, что у тебя в модальном окне появится форма с инпутами и ты их не будешь видеть.

msmirnov commented on index.css:
```diff
@@ -0,0 +1,312 @@
+    display: inline-block;
+}
+
+img
```
Для таких распространённых тегов, как img и input, лучше не писать стили. Надо на них навешать классы и писать стили уже для конкретного класса. Так проще в будущем поддерживать код и будет меньше багов.

msmirnov commented on index.css:
```diff
@@ -0,0 +1,312 @@
+
+.carousel
+{
+    text-align: left;
```
Это станет не нужно, когда ты уберешь text-align с body.

msmirnov commented on index.css:
```diff
@@ -0,0 +1,312 @@
+    text-align: left;
+    margin: auto;
+    margin-top: 10%;
+    overflow: hidden;
```
Не понял, зачем

Kuvadorian replied:
Насчёт overflow: hidden, оставил т.к если я уберу его, то страница станет шириной в 3000px, которые будут занимать все 10 картинок, а так я скрываю картинки, которые пользователь пока видеть не должен

msmirnov replied:
Ок, понял

msmirnov commented on index.css:
```diff
@@ -0,0 +1,312 @@
+.carousel
+{
+    text-align: left;
+    margin: auto;
```
Зачем?

msmirnov commented on index.css:
```diff
@@ -0,0 +1,312 @@
+.slide-to-black-out
+{
+    background: rgba(0, 0, 0, .5);
+    color: #fff;
```
Где-то цвета заданы словами, где-то кодами. Нужно, чтобы было одинаково. Вариант #fff сильно лучше, чем white.

msmirnov reviewed (changes requested):
Модальное окно надо переделать, чтобы оно было похоже на макет. От текущей его реализации у меня появилась резкая боль в глазах :)

🍅

Kuvadorian committed:
Учёл замечания

Насчёт overflow: hidden, оставил т.к если я уберу его, то страница станет шириной в 3000px, которые будут занимать все 10 картинок, а так я скрываю картинки, которые пользователь пока видеть не должен

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

msmirnov commented:
Всё ещё сильно не похоже на макет: https://user-images.githubusercontent.com/5352441/47854735-53163580-de04-11e8-8ec2-5d7dfafded25.png

🍅

Kuvadorian committed:
Переделал модальное окно

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

Kuvadorian committed:
Поправил модальное окно(2)

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты
